Add SDK v5 integration steps (#794)
adjust/dev-docs: src/content/docs/sdk/ios/features/skad.mdx

title: Set up SKAdNetwork and conversion values
description: Configure SKAdNetwork features for your iOS apps.
slug: en/sdk/ios/features/skad
multiVersion: true
---

<SdkVersion version="v5">

<Callout type="important">

This feature is only available on devices running iOS 14 and above.

</Callout>

StoreKit Ad Network (SKAdNetwork) is Apple's attribution framework for app install and reinstall attribution. The SKAdNetwork workflow goes like this:

1. Apple gathers attribution information and notifies the relevant ad network.
2. The network sends a postback with this information to Adjust.
3. Adjust displays SKAdNetwork data in [Datascape](https://help.adjust.com/en/suite/article/datascape).

## [Disable SKAdNetwork communication](disable-skadnetwork-communication)

<CodeBlock title="Method signature">

```objc

> - (void)deactivateSKAdNetworkHandling;

Reviewer: ```suggestion
- (void)disableSkanAttribution;
```

```

</CodeBlock>

The Adjust SDK communicates with SKAdNetwork by default. The SDK registers for SKAdNetwork attribution upon initialization.

> Your config object contains a boolean `isSKAdNetworkHandlingActive` property that controls this behavior. You can disable SKAdNetwork communication by calling the `deactivateSKAdNetworkHandling` method with no argument.

Reviewer: ```suggestion
You can disable SKAdNetwork communication by calling the `disableSkanAttribution ` method with no argument.
```

<Callout type="important">

> You must call the `deactivateSKAdNetworkHandling` method _before_ initializing the Adjust SDK.

Reviewer: ```suggestion
You must call the `disableSkanAttribution` method _before_ initializing the Adjust SDK.
```

</Callout>

<Tabs>
<Tab title="Swift" sync="swift">

```swift
let yourAppToken = "{YourAppToken}"
let environment = ADJEnvironmentSandbox
let adjustConfig = ADJConfig(
    appToken: yourAppToken,
    andEnvironment: environment)
//...

> adjustConfig.deactivateSKAdNetworkHandling()

Reviewer: ```suggestion
adjustConfig.disableSkanAttribution()
```

//...

> Adjust.appDidLaunch(adjustConfig)

Reviewer: ```suggestion
Adjust.initSdk(adjustConfig)
```

```

</Tab>
<Tab title="Objective-C" sync="objc">

```objective-c
ADJConfig *adjustConfig = [[ADJConfig alloc] initWithAppToken:@"{YourAppToken}"
                                                   andEnvironment:ADJEnvironmentSandbox];

> [adjustConfig.deactivateSKAdNetworkHandling];

Reviewer: ```suggestion
[adjustConfig disableSkanAttribution];
```

/// ...

> [Adjust appDidLaunch:adjustConfig];

Reviewer: ```suggestion
[Adjust initSdk:adjustConfig];
```
